### DISCUSSION TITLE
[docs]: what versions are currently supported wrt security patches?

### DISCUSSION THREAD

**Discussion by artificial-intelligence** [containers/podman#19946] [Q&A]
### Issue Description

Could you please provide some information somewhere on what podman releases are supported by the project itself? That is, which branches get security updates?

I didn't find it:

- in the user / installation documentation on podman.io
- in the security.md of this repository (which links to the containers/common/security.md)
- I can look at active git branches, but I think it should be more discoverable for end users which release are supported and which versions are deprecated.

Thanks in advance.

### Steps to reproduce the issue

Steps to reproduce the issue
1.
2.
3.

### Describe the results you received

Describe the results you received

### Describe the results you expected

Describe the results you expected

### podman info output

```yaml
If you are unable to run podman info for any reason, please provide the podman version, operating system and its version and the architecture you are running.
```

### Podman in a container

No

### Privileged Or Rootless

None

### Upstream Latest Release

Yes

### Additional environment details

Additional environment details

### Additional information

Additional information like issue happens only occasionally or issue happens with a particular architecture or on a particular setting

**Comment by artificial-intelligence** [comment]
I'm writing this request, because we currently try to decide in the openstack kolla-ansible project which podman version we should support initially.

our project strives to build production ready containers of openstack and also deploys them.

So as we are introducing podman (currently only supporting docker) I'd like to know what release is supported for what time frame.

This is further complicated by the fact that we support multiple Host Distros, like rocky linux, ubuntu and debian.

See also the change discussion upstream, if you are interested:

https://review.opendev.org/c/openstack/ansible-collection-kolla/+/852240

You can find other related patches in openstack kolla for podman support via this link, if you are interested: https://review.opendev.org/q/topic:kolla-podman

**Comment by Luap99** [comment]
We only support the latest released version, so 4.6.2 at the moment. If distros ship older versions then the distro has to support that and backport security fixes when required.

There are some -rhel branches in the repo which are used for versions in RHEL. But these contain only RHEL approved backports which can include features so not just security fixes. And they also are EOL once the RHEL release no longer supports that version so I wouldn't recommend relying on them either.

**Comment by afbjorklund** [comment]
This matrix is somewhat useful: https://repology.org/project/podman/badges

Repository | Podman
--- | ---
AlmaLinux 8 | 4.4.1
AlmaLinux 9 | 4.4.1
CentOS 7 | 1.6.4
CentOS 8 | 3.3.1
CentOS Stream 8 | 4.6.1
CentOS Stream 9 | 4.6.1
Debian 11 | 3.0.1
Debian 12 | 4.3.1
Fedora 37 | 4.6.1
Fedora 38 | 4.6.2
Ubuntu 20.04 | 3.4.2*
Ubuntu 22.04 | 3.4.4

\* Kubic stable
